lib/vrl/stdlib/src/lib.rs

```diff
@@ -488,8 +492,10 @@ pub fn all() -> Vec<Box<dyn vrl::Function>> {
         Box::new(Log),
         #[cfg(feature = "match")]
         Box::new(Match),
-        #[cfg(feature = "match")]
+        #[cfg(feature = "match_any")]
```

Comment:
Not strictly related, but this seemed like a mismatch.

Reply:
Yeah, good catch.